docs/apis/GraphQL-Tutorial.md

```diff
@@ -90,6 +90,9 @@ Most people want to get routing results out of OTP, so lets see the query for th
         itineraries {
             startTime
             endTime
+            emissionsPerPerson {
```

Comment:
Since the `plan.graphql` is used in the tutorial, can you please create another one just with the emissions?

Reply:
Of course, I should have realised.

Reply:
Actually, there are a few other plan query tests in the same folder and if you like, you can just add it to another one, for example `plan-extended.graphl`.